src/main/java/io/stargate/sgv2/jsonapi/service/schema/collections/CollectionSchemaObject.java

```diff
@@ -326,7 +250,8 @@ private static CollectionSchemaObject createCollectionSettings(
         switch (collectionNode.get(TableCommentConstants.SCHEMA_VERSION_KEY).asInt()) {
           case 1:
             return new CollectionSettingsV1Reader()
-                .readCollectionSettings(collectionNode, keyspaceName, collectionName, objectMapper);
+                .readCollectionSettings(
```

Comment:
Same here, aren't `keyspaceName` and `collectionName` accessible from `tableMetadata`?

Reply:
same as above, unpicking all the diff ways it was created felt like a large job. created https://github.com/stargate/data-api/issues/1459 to do that